### PR TITLE
Survey page calculation

### LINKED ISSUE #39: User Story: in SurveyPage, selecting a style choice enables "Next Page"/"Overview" button
As a user, I should be unable to click the "Next Page"/"Overview" button until a style has been selected.

### LINKED ISSUE #42: User Story: In SurveyPage, clicking the "Overview" button will route to SurveyOverview
As a user on the final SurveyPage, clicking the "Overview" button should bring me to the Overview page

### LINKED ISSUE #49: User Story: When Editing a selection, the "Overview" button is displayed instead of "Next Page"
As a user, when editing a survey selection in the survey page after clicking "Edit" from the overview page, I should see the "Overview" button instead of the "Next Page" button. 
I should not have to go through the whole survey again, and rather be able to navigate to the overview page after editing a selection.

### LINKED ISSUE #52: Create surveypage.js route file in web/src/setup/routes

### LINKED ISSUE #54: Create surveyresults.js route file in web/src/setup/routes

### LINKED ISSUE #58: Add redux devtools dependencies

### PR DESCRIPTION
**Pull request by GreyMatteOr**
### This PRs additions:

- Survey page introduces dynamic routing and button disable / enable logic depending on selections. 
- When a selection has been made for every page, "Next" changes to "see results" and button routes to the "/results" page instead of the next survey.

 
### Reviewer starting point: 

Spin up a local copy. 
Navigate to `localhost:3000/survey/1` to start the survey.
 
### How to manually test:  

In browser only, no tests yet.

 
### Relevant tickets: 

closes #58 
closes #54 
closes #52 
closes #49
closes #42 
closes #39 
 
### Screenshots:
Next Disabled
Next Enabled
Final Page Routing

### CODE REVIEW

**Review by MichaelEWalker87 — COMMENTED** [review]
Everything looks good on my end thanks for your hard work

File: code/api/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "setup:db": "npm run db:migrate && npm run db:seed",
     "db:migrate": "./node_modules/.bin/sequelize db:migrate",
     "db:seed": "./node_modules/.bin/sequelize db:seed:all",
-    "start": "`nodemon` src/index.js --exec babel-node",
```
Comment (bretm9) [inline code/api/package.json:12]: Thanks for this fix. This gets the backend back up and running

File: code/web/src/modules/survey/Survey/Item.js [hunk 2]
```diff
@@ -28,9 +28,8 @@ class Item extends PureComponent {
   }
 
   handleItemClick = () => {
-    console.log(this.state.selected)
     this.setState({ selected: !this.state.selected });
-    updateSelectedItems(this.props.item, this.state.selected)
+    this.props.updateSelectedItems(this.props.item, !this.state.selected, this.props.page)
```
Comment (bretm9) [inline code/web/src/modules/survey/Survey/Item.js:32]: this is a critical piece to getting updateSelectedItems working with Redux. Thanks

**Review by bretm9 — COMMENTED** [review]
All looks good to me! Approved.

**Pull request merged**